genie/input_to_database.py

```diff
@@ -250,7 +253,7 @@ def validatefile(syn, entities, validation_statusdf, error_trackerdf,
     file_users = [entities[0].modifiedBy, entities[0].createdBy]
 
     check_file_status = check_existing_file_status(
-        validation_statusdf, error_trackerdf, entities)
+        validation_status_table, error_tracker_table, entities)
```

Comment:
Any reason you switch to using `table` instead of `df`?

Reply:
I noted in the issue description that some fxns use one and some use the other. I made it standard.